### DISCUSSION TITLE
Private modified repo also getting banned

### DISCUSSION THREAD

**Discussion by infographyc** [ratsan/Student-Database#5] [Q&A] [answered]
As a user said heroku is banning flagged codes or repos. I tried to modify the code and push in heroku git. My account got suspended instantly after push. So, is this an end for this project?

**Comment by codehasan** [comment, marked as answer]
Yes. It's much likely an end until the dev makes any changes to fix it. However as I said you should look up to other repos, userge will be a good choice.

**Reply by onxx-x146** [reply]
Gh
